pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -13,6 +13,20 @@ done
 # Install or update the guestagent binary
 install -m 755 "${LIMA_CIDATA_MNT}"/lima-guestagent /usr/local/bin/lima-guestagent
 
+# Set up Environment Variable to be loaded via profile
+if [[ ! -e /etc/profile.d/lima.sh ]]; then
+    touch /etc/profile.d/lima.sh
+	echo "#! /bin/sh -" > /etc/profile.d/lima.sh
+fi
+
+for f in $(seq 0 $((LIMA_CIDATA_ENVS - 1))); do
+	keyvar="LIMA_CIDATA_ENVS_${f}_KEY"
+	valuevar="LIMA_CIDATA_ENVS_${f}_VALUE"
+    key="$(eval echo \$"$keyvar")"
+    value="$(eval echo \$"$valuevar")"
+    echo "export "$key"="$value"" >> test.txt
```

Comment:
File name seems wrong

Reply:
yup little sleep and copy and paste error